src/js/visual/configurator-csr.js

```diff
@@ -949,6 +955,13 @@ ripe.ConfiguratorCsr.prototype._destroyInitialsResources = function() {
     this._deinitCsrRenderedInitials();
 };
 
+/**
+ * Completely cleanup and destroy CSR config handlers.
+ */
+ripe.ConfiguratorCsr.prototype._destroyConfig = function() {
+    this._unregisterConfigGlobalHandlers();
+};
+

```

Comment:
This doesn't make sense to exist as a `destroy` method as the context is registering and unregistering handlers.
Destroy methods (in this configurator context) are methods to completly self handle the release and destruction of a module of the configurator. Unregistering handlers is just one step of that.

The destroy method would make sense to exist if/when it is a self containing stuff. In this case the configurator `deinit()` is the so called `destroyConfig()` in this context